erc721-meta-transaction/src/index.ts

```diff
@@ -0,0 +1,48 @@
+import { Chains, Token, Wallet } from './models';
+import { User }                  from './user';
+import { Payer }                 from './payer';
+
+export function main() {
+    testMetaTransactionNftTransfer()
+        .then(() => console.log('done'));
+}
+
+async function testMetaTransactionNftTransfer() {
+    const user = new User();
+    const payer = new Payer();
```

Comment:
Not sure that these are the concepts/names that are used in the documentation but make sure you align it with Abdullah so that the documentation and the example use the same terminology